torch/cuda/__init__.py

```diff
@@ -322,40 +323,83 @@ def can_device_access_peer(device: _device_t, peer_device: _device_t) -> bool:
     return torch._C._cuda_canDeviceAccessPeer(device, peer_device)
 
 
-@contextlib.contextmanager
-def stream(stream):
+class StreamContext(object):
     r"""Context-manager that selects a given stream.
 
     All CUDA kernels queued within its context will be enqueued on a selected
     stream.
 
     Args:
+        Stream (Stream): selected stream. This manager is a no-op if it's
+            ``None``.
+
+    .. note:: Streams are per-device.
+    """
+    cur_stream : Optional['torch.classes.cuda.Stream']
```

Comment:
Yeah, so I have tried alias-ing it, I keep running into this error 
```
 AttributeError: partially initialized module 'torch' has no attribute 'jit' 
```
So I tried adding it here in the `StreamContext` and also outside it. Both resulted in this error.

Reply:
Let's add custom resolution logic for these classes, just to see what the end result is like. That is, let's modify the type parsing logic to map `torch.cuda.Stream` to `torch.classes.cuda.Stream` (and something similar for `Event`) since we can't use a type alias.

Reply:
Done. `frontend.py` now goes though the sourcelines and replaces `torch.cuda.Stream` to `torch.classes.cuda.Stream` and `torch.cuda.Event` to `torch.classes.cuda.Event`